cmd/ctr/commands/run/run_windows.go

```diff
@@ -149,3 +157,14 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 func getNewTaskOpts(_ *cli.Context) []containerd.NewTaskOpts {
 	return nil
 }
+
+func getNetNsPath(ctx gocontext.Context, t containerd.Task) (string, error) {
```

Comment:
nit: Should be `GetNetNSPath`

Reply:
changed

Reply:
Sorry, I typo'd my comment. I meant that `NS` should be capitalized. I accidentally capitalized `Get` as well. Think this should just be `getNetNSPath`.